web-app/django/VIM/apps/instruments/models/instrument_name.py

```diff
@@ -8,3 +8,9 @@ class InstrumentName(models.Model):
     source_name = models.CharField(
         max_length=50, blank=False, help_text="Who or what called the instrument this?"
     )  # Stand-in for source data; format TBD
+    description = models.CharField(
+        blank=True, help_text="Description of the instrument name"
+    )  # Stand-in for description
+    alias = models.CharField(
```

Comment:
What is the difference between an alias and an instrument name?

Reply:
Instrument name corresponds to "label" column while alias corresponds to the "also known as" column in wikidata.
![image](https://github.com/user-attachments/assets/3ab2d4ff-cb66-4ae4-bdc1-547d034c82dd)

Reply:
But aren't those just names? And aren't there multiple? I'm not sure why these wouldn't be stored in our database as additional instances of an instrument name.

Reply:
This makes a lot of sense. I should do it as you say.